src/style.css

```diff
@@ -1,3 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+
+  --activation-color: #00ACDC;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  margin: 10px 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding:0;
+}
+
+.nav__item {
+  margin: 0;
+  padding: 0 10px;
+}
+
+.nav__item:first-child {
+  padding-left: 0;
+}
+
+.nav__item:last-child {
+  padding-right: 0;
+}
+
+.nav__link {
+  position: relative;
+  text-transform: uppercase;
+  display: inline-block;
+  font-size: 12px;
+  padding: 0;
+  line-height: 60px;
+  height: 60px;
```

Comment:
can be removed

Reply:
if you delete it, the font is not the right size. the blue bar changes position.